TankDriver.App/TankDriver.App.csproj

```diff
@@ -94,4 +95,4 @@
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
The changes here looks terrible, but I can't see any current problems with them. It seems that MonoDevelop have made this file a little better. I'll need to check that with Visual Studio later.